### PR TITLE
Updates for code of conduct and security section

### PR DESCRIPTION
**Pull request by abstractj**
As part of the CNCF onboarding process, projects are required to explicitly state the CNCF Code of Conduct in their README files. This is aligned with CNCF's standards, which emphasize inclusive and respectful collaboration in project communities.

Added to that, the security section was updated to remain aligned with https://www.keycloak.org/security.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -6,18 +6,13 @@ vulnerabilities, as well as suggestions around hardening the software and our pr
 
 ## Reporting a suspected vulnerability
 
-It is important that suspected vulnerabilities are disclosed in a responsible way, and are not publicly disclosed until
-after they have been analysed and a fix is available.
+It is important that suspected vulnerabilities are disclosed in a responsible way, and are not publicly disclosed until after they have been analysed and a fix is available.
```
Comment (jonkoops) [inline SECURITY.md:9]: Should we mention that automated security scan results and public CVEs in dependencies are not considered a security issue? They are pretty common to get filed.

**Review by jonkoops — APPROVED** [review]

File: SECURITY.md (hunk 1 above)
Comment (abstractj) [reply SECURITY.md:9]: @jonkoops I believe this is covered by "If you are reporting known CVEs related to third-party libraries used in Keycloak, please [create a new GitHub issue](https://github.com/keycloak/keycloak/issues/new/choose)."

File: SECURITY.md [hunk 2]
```diff
@@ -6,18 +6,13 @@ vulnerabilities, as well as suggestions around hardening the software and our pr
...
 
-To report a security vulnerability, send an email to keycloak-security@googlegroups.com.
+To report a security vulnerability in the Keycloak codebase, send an email to keycloak-security@googlegroups.com. Please include the version affected, provide detailed instructions on how to reproduce the issue, and include your contact information for acknowledgements. If you are reporting known CVEs related to third-party libraries used in Keycloak, please create a new GitHub issue.
```
Comment (vmuzikar) [inline SECURITY.md]:
```suggestion
To report a security vulnerability in the Keycloak codebase, send an email to keycloak-security@googlegroups.com. Please include the version affected, provide detailed instructions on how to reproduce the issue, and include your contact information for acknowledgements. If you are reporting known CVEs related to third-party libraries used in Keycloak, please create a new GitHub issue, no need to send the email.
```

File: SECURITY.md (hunk 1 above)
Comment (jonkoops) [reply SECURITY.md:9]: Sounds good

**Comment by jonkoops** [comment]
@abstractj might want to sign off on your commit, looks like the DCO also applies here.

File: SECURITY.md (hunk 2 above)
Comment (abstractj) [reply SECURITY.md]: Thanks @vmuzikar for the suggestion. We're trying to stay in line with what's on the Keycloak security page (https://www.keycloak.org/security).

If it's ok for you, let's stick with what we've got for now. If we start getting more emails like this, we can make it clear for people submitting it.

**Review by jonkoops — APPROVED** [review]

File: SECURITY.md (hunk 2 above)
Comment (vmuzikar) [reply SECURITY.md]: Ok, works for me. Thanks.

**Review by vmuzikar — APPROVED** [review]

**Pull request merged**